src/main/resources/performance-scripts/create_audit_logs_flow.jmx

```diff
@@ -0,0 +1,11988 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Please move this folder outside of resources folder to not include into deployment artifact.
You can move this to the root folders